x-pack/plugins/maps/server/routes.js

```diff
@@ -30,7 +30,10 @@ export function initRoutes(server, licenseUid) {
     handler: async (request) => {
 
       if (!request.query.id) {
-        server.log('warning', 'Must supply id parameters to retrieve EMS file');
+        server.log('warning',
```

Comment:
I believe you forgot this line here; We dont translate server logs or any server strings for that matter.